### PR TITLE
footer-update

### PR DESCRIPTION
**Pull request by sihameelati**
### Jira Tickets
https://dmndev.atlassian.net/browse/CMS-7399

### Testing

- Clone this repo if it's the first time.
- Checkout this branch
`checkout feature/footer-update`

- Install all packages
`npm i`
- Build Code
`npm run build`
- Run Server
`npm run start`
Open a browser tab and navigate to ...
[localhost](http://localhost:8080/)

now make sure the footer look like this
Screen Shot 2023-06-15 at 4 29 00 PM

And the links should be as follows:

Advertise With Us | https://mediumgiant.co/advertise-with-the-dallas-morning-news/ 

Autos | https://www.dallasnews.com/business/autos/ 

Celebrations | https://www.legacy.com/celebration/dallasmorningnews/  

Classifieds | https://www.dallasnews.com/classifieds/ 

Jobs | https://dfwishiring.dallasnews.com/?_gl=1*5zjw31*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjAyLjU2LjAuMA.. 

Obituaries | https://obits.dallasnews.com/obituaries/dallasmorningnews?_gl=1*41rete*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjE1MDY5MS40MjUuMS4xNjg2MTU2MDQxLjU5LjAuMA.. 

Public Notices | https://texaspublicnotices.com/

Buy | http://store.dallasnews.com/?_gl=1*5zjw31*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjAyLjU2LjAuMA.. 

Archive | https://archive.dallasnews.com/?_gl=1*5zjw31*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjAyLjU2LjAuMA.. 

Back copies | https://store.dallasnews.com/collections/back-copies?_gl=1*hv9w8c*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA.. 

Licensing | https://archive.dallasnews.com/CS.aspx?VP3=CMS3&VF=ContactUsStatic_VForm&_gl=1*hv9w8c*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA.. 

Photo reprints | https://store.dallasnews.com/pages/photo-request-form?_gl=1*btxa9n*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA.. 

Purchase today’s paper | https://finddallasnews.com/?_gl=1*btxa9n*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA.. 

Company | http://dmnmedia.com/about-us/company-info 

About The Dallas Morning News | https://www.dallasnews.com/help/about-us/ 

Contact Us | https://dallasnews.zendesk.com/hc/en-us/categories/360002750811-Contact-The-Dallas-Morning-News

Careers | https://dallasnewscorporation.mua.hrdepartment.com/hr/ats/JobSearch/viewAll 

FAQs | https://www.dallasnews.com/help/ 

Customer Support |  

ePaper | https://epaper.dallasnews.com/?_gl=1*btxa9n*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA.. 

ePaper (Al Día) | https://epaper.aldiadallas.com/?_gl=1*btxa9n*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA.. 

Help and feedback | https://www.dallasnews.com/help/ 

Podcasts | https://www.dallasnews.com/topic/podcasts/ 

Manage your account | https://myaccount.dallasnews.com/signin.html?_gl=1*15lks76*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA.. 

Newsletters | https://www.dallasnews.com/newsletters/

Newspaper subscription | https://dallasnews.zendesk.com/hc/en-us/categories/360002559932-Print-account-paper-delivery 

Vacation stop/Make a payment | https://myaccount.dallasnews.com/?_gl=1*15lks76*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA..

Website Support | https://docs.google.com/forms/d/e/1FAIpQLSc5q1vcwEtdtLR3IbASfx27PlFjfZ6YA1UImgTyN7ZQKTfWJw/viewform

Content Removal Review | https://www.dallasnews.com/contentremovalreview/ 

Do Not Sell My Personal Information | https://www.dallasnews.com/help/privacy-policy/#manageConsent 

Privacy Policy | https://www.dallasnews.com/help/privacy-policy/ 

Site Map | https://www.dallasnews.com/sitemap/

Terms of Service | https://www.dallasnews.com/help/terms-of-service/

- We change the copyright as well from 2022 to 2023

### CODE REVIEW

**Review by jte1344 — APPROVED** [review]
LGTM! All links appear to be correct!

File: src/footer/footer.html [hunk 1]
```diff
@@ -135,208 +84,149 @@
...
         </li>
-        <li
-          class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block"
-        >
-          <a href="https://dallasnewscorporation.mua.hrdepartment.com/hr/ats/JobSearch/viewAll"
-            >Careers</a
-          >
+        <li class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block">
+          <a
+            href="https://obits.dallasnews.com/obituaries/dallasmorningnews?_gl=1*41rete*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjE1MDY5MS40MjUuMS4xNjg2MTU2MDQxLjU5LjAuMA..">Obituaries</a>
```
Comment (rlsmileg) [inline src/footer/footer.html:106]: remove the _gl and _ga params please. @sihameelati They were likely added by the web browser automatically.

File: src/footer/footer.html [hunk 2]
```diff
@@ -135,208 +84,149 @@
...
+        <li class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block">
+          <a href="https://texaspublicnotices.com/">Public Notices</a>
         </li>
       </ul>
     </div>
     <div class="column small-12 medium-12 large-expand">
       <div class="font-medium block uppercase hover_underline pb-6">
-        <a href="https://www.belomediagroup.com">Advertise With Us</a>
+        <a
+          href="http://store.dallasnews.com/?_gl=1*5zjw31*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjAyLjU2LjAuMA..">Buy</a>
```
Comment (rlsmileg) [inline src/footer/footer.html:116]: remove the _gl and _ga params please. @sihameelati They were likely added by the web browser automatically.

File: src/footer/footer.html [hunk 3]
```diff
@@ -135,208 +84,149 @@
...
+          <a
+            href="https://archive.dallasnews.com/?_gl=1*5zjw31*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjAyLjU2LjAuMA..">Archive</a>
         </li>
-        <li
-          class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block"
-        >
-          <a href="https://www.dallasnews.com/classifieds/">Classifieds</a>
+        <li class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block">
+          <a
+            href="https://store.dallasnews.com/collections/back-copies?_gl=1*hv9w8c*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA..">Back
```
Comment (rlsmileg) [inline src/footer/footer.html:126]: remove all _gl or _ga params. They are tracking params that should not have been added here.

File: src/footer/footer.html [hunk 4]
```diff
@@ -135,208 +84,149 @@
...
+            copies</a>
         </li>
-        <li
-          class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block"
-        >
-          <a href="https://dfwishiring.dallasnews.com">Jobs</a>
+        <li class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block">
+          <a
+            href="https://archive.dallasnews.com/CS.aspx?VP3=CMS3&VF=ContactUsStatic_VForm&_gl=1*hv9w8c*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA..">Licensing</a>
```
Comment (rlsmileg) [inline src/footer/footer.html:131]: remove all _gl or _ga params. They are tracking params that should not have been added here.

File: src/footer/footer.html [hunk 5]
```diff
@@ -135,208 +84,149 @@
...
         </li>
-        <li
-          class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block"
-        >
-          <a href="http://obits.dallasnews.com/obituaries/dallasmorningnews"
-            >Obituaries</a
-          >
+        <li class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block">
+          <a
+            href="https://store.dallasnews.com/pages/photo-request-form?_gl=1*btxa9n*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA..">Photo
```
Comment (rlsmileg) [inline src/footer/footer.html:135]: remove all _gl or _ga params. They are tracking params that should not have been added here.

File: src/footer/footer.html [hunk 6]
```diff
@@ -135,208 +84,149 @@
...
         </li>
-        <li
-          class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block"
-        >
-          <a href="https://myaccount.dallasnews.com/signin.html"
-            >Manage your account</a
-          >
+        <li class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block">
+          <a
+            href="https://epaper.aldiadallas.com/?_gl=1*btxa9n*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA..">ePaper
```
Comment (rlsmileg) [inline src/footer/footer.html:179]: remove all _gl or _ga params. They are tracking params that should not have been added here.

File: src/footer/footer.html [hunk 7]
```diff
@@ -135,208 +84,149 @@
...
+        <li class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block">
+          <a href="https://www.dallasnews.com/topic/podcasts/">Podcasts</a>
         </li>
-        <li
-          class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block"
-        >
-          <a href="https://epaper.dallasnews.com">ePaper</a>
+        <li class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block">
+          <a
+            href="https://myaccount.dallasnews.com/signin.html?_gl=1*15lks76*_ga*ODgzNTExNjIxLjE2NTM1MTIxNTU.*_ga_8T538V93BE*MTY4NjEwODc2OC40MjQuMS4xNjg2MTA5MjQ0LjE0LjAuMA..">Manage
```
Comment (rlsmileg) [inline src/footer/footer.html:190]: remove all _gl or _ga params. They are tracking params that should not have been added here.

**Pull request merged**